_data/hosting_providers.json

```diff
@@ -1371,8 +1371,8 @@
   },
   {
     "name": "one.com",
-    "link": "",
-    "category": "partial",
+    "link": "https://www.one.com/",
+    "category": "full",
     "tutorial": "",
     "announcement": "",
```

Comment:
Can you fill this announcement link? Anything, like a blog post, documentation, etc, that publicly acknowledges your automatic support for HTTPS.

Reply:
Added date, don't think we have any announcement link (I work there), before we had a service where the customer could enable comodo certificate manually but when we switched to letsencrypt we silently just enabled it for everyone.